### PR TITLE
Remove /api/ from relative paths across the code

### LINKED ISSUE #12: Fix the bug where api does not run itself automatically on AWS
`run_api.sh` is not executed on AWS. We need to identify the reason and correct it

### PR DESCRIPTION
**Pull request by adrian-golian**
Closes #12

### CODE REVIEW

File: api/game_routines.R [hunk 1]
```diff
@@ -1,8 +1,8 @@
-indexation <- read_csv("./api/indexation.csv")
+indexation <- read_csv("indexation.csv")
 
 get_path <- function(game_uuid, r = -1) {
-  if (r == -1) return(paste0("./game_data/", game_uuid, ".RDS"))
-  if (r >= 0) return(paste0("./game_data/", game_uuid, "_", r, ".RDS"))
+  if (r == -1) return(paste0("../game_data/", game_uuid, ".RDS"))
```
Comment (adrian-golian) [inline api/game_routines.R:4]: If the api dir is the new root then this dir (`game_data`) is not ../ is it?

Comment (maciej-pomykala) [reply api/game_routines.R:4]: I thought of `game_data` as something on the same level as `api`

Comment (adrian-golian) [reply api/game_routines.R:4]: I'm saying there is nothing above `api` - that's the deployment directory.

**Comment by adrian-golian** [comment]
#12 is closed, so closing this PR.

**Pull request closed without merging**